### PR TITLE
K8s 1.6 upgrade

### PR DESCRIPTION
**Pull request by dkoshkin**
TODO use prod repos and k8s 1.6 when it comes out.

### CODE REVIEW

File: ansible/roles/calico/templates/calico.yaml [hunk 1]
```diff
@@ -11,29 +11,6 @@ data:
   # Configure the Calico backend to use.
   calico_backend: "bird"
 
-  # The CNI network configuration to install on each node.
```
Comment (alexbrand) [inline ansible/roles/calico/templates/calico.yaml]: Let's document why we are not putting this in the ConfigMap. I am sure we'll ask this question in the future. A comment in the PR description should work.

File: pkg/cli/upgrade.go [hunk 2]
```diff
@@ -320,6 +320,14 @@ func upgradeNodes(out io.Writer, plan install.Plan, opts upgradeOpts, nodesNeedU
 		}
 	}
 
+	// Migate Kubernetes etcd to v3
+	// TODO in future KET releases need to check for a minimum version
+	// All KET releases with v1.6+ do not need this to run
+	util.PrintHeader(out, "Migrate: Kubernetes Etcd Cluster", '=')
```
Comment (alexbrand) [inline pkg/cli/upgrade.go]: Will this run again if the etcd nodes have already been migrated?

Comment (dkoshkin) [inline pkg/cli/upgrade.go]: It would but in my testing rerunning the migration was a noop

**Comment by dkoshkin** [comment]
Moved CNI config out to an Ansible task from a ConfigMap. It is not possible to support rolling and to control when the config gets updated on each node individually when using a single ConfgiMap that all pods read from.

**Pull request merged**